beacon-chain/rpc/prysm/v1alpha1/validator/proposer_empty_block_test.go

```diff
@@ -18,6 +18,7 @@ func Test_getEmptyBlock(t *testing.T) {
 	config.BellatrixForkEpoch = 2
 	config.CapellaForkEpoch = 3
 	config.DenebForkEpoch = 4
+	config.ElectraForkEpoch = 5
```

Comment:
Can you document the effect of setting these fork epoch values? I assume the point here is just to set the highest epoch to something that won't overflow (when it is set to max uint) and making sure these epochs don't conflict with each other.

Reply:
This is just so we can test the conditions.  Here is how it's used in the test case:
`slot: primitives.Slot(params.BeaconConfig().ElectraForkEpoch) * params.BeaconConfig().SlotsPerEpoch,`
which gets passed to `getEmptyBlock(tt.slot)`, then reached under `case epoch >= params.BeaconConfig().ElectraForkEpoch:`